modules/dnn/src/layers/eltwise_layer.cpp

```diff
@@ -156,9 +156,15 @@ class EltwiseLayerImpl CV_FINAL : public EltwiseLayer
             p.dst = &dst;
             p.op = op;
             p.nstripes = nstripes;
-            p.channels = (dst.dims == 4 ? dst.size[1] : 1);
-            p.planeSize = (dst.dims >= 3 ? dst.size[dst.dims - 1] * dst.size[dst.dims - 2] :
-                                           dst.size[dst.dims - 1]);
+            p.channels = (dst.dims >= 4 ? dst.size[1] : 1);
+
+            switch (dst.dims) {
+                case 3:  p.planeSize = dst.size[1] * dst.size[2];               break;
+                case 4:  p.planeSize = dst.size[2] * dst.size[3];               break;
+                case 5:  p.planeSize = dst.size[2] * dst.size[3] * dst.size[4]; break;
+                default: p.planeSize = dst.size[dst.dims - 1];
```

Comment:
What about `total` method from https://github.com/opencv/opencv/blob/master/modules/dnn/include/opencv2/dnn/shape_utils.hpp? We just need to multiply all the dimensions starts from 2.

Reply:
if `dst.dims = 3` we multiply 1 and 2 dimensions.

Reply:
Maybe `p.planeSize = dst.total(1) / p.channels`?

Reply:
@l-bat, I think that `p.planeSize = dst.total(dst.dims >= 4 ? 2 : 1)` is better.